### DISCUSSION TITLE
SSH access

### DISCUSSION THREAD

**Discussion by javatechno** [jbangdev/jbang#1785] [Q&A] [answered]
I'm sorry if the question was already answered already, but how can I access a repository via git@ protocol?
Also, is there an instruction about building custom JBang-Camel Docker image from source?

**Comment by maxandersen** [comment, marked as answer]
You mean as in accessing a .java or .jar file using git Ssh protocol?

That is not currently supported. Not sure it ever will be. It would require us to clone the project before accessing the files. With http/Https we aren't really accessing using git but a site that just coincidentally serves out git based content.

About camel and jbang - better to ask that in camel forums.
